src/devices/honeywell_cm921.c

```diff
@@ -0,0 +1,449 @@
+/** @file
+    Honeywell CM921 Thermostat
+
+    Copyright (C) 2020
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+/**
+Honeywell CM921 Thermostat.
+
+868Mhz FSK, PCM, Start/Stop bits, reversed, Manchester.
+*/
+
+#include <stdlib.h>
+#include <mongoose.h>
```

Comment:
Both these should not be needed,

Reply:
Mongoose is used for `cs_to_hex()`, though actually `hex()` shouldn't be there, it's used for `DATA_STRING, packet_hex,` but please either debug print buffers or output decoded fields.
On that, can you document the packet field structure in a doc comment to `honeywell_cm921_decode()`? I guess there are different packets looking like the struct's?

Reply:
Yeah, I got rid of these, the various hex conversion were redundant anyway. 

The packet field structure is the Evohome structure, there are a couple links in the code that point to webpages that describe some of it. The full specification is not known.